javascript/javascript1/week1/Ez Namey/script.js

```diff
@@ -0,0 +1,27 @@
+const firstWords = ["Easy", "Amazing", "Corporate", "Unique", "Beautiful", "Masive", "Ordinary", "New", "Old", "Soft"];
+
+const similarWords = ["Simple", "Awesome", "Business", "Special", "Elegant", "Huge", "Common", "Fresh", "Ancient", "Smooth"];
+
+const randomNumber = Math.floor(Math.random() * 10);
+const randomNumber2 = Math.floor(Math.random() * 10);
+
+let startupName =  firstWords[randomNumber] + " " + similarWords[randomNumber2];
+
+console.log(startupName);
```

Comment:
The output have to be the name of a startup and the number of characters in it, like in the example: "The startup: "Easy Corporation" contains 16 characters"

Reply:
I fixed alredy, thank youu so much =D